regulations/static/regulations/css/less/typography.less

```diff
@@ -42,35 +42,52 @@ h3,
 h4,
 h5,
 h6 {
-    .font-bold;
-    margin-bottom: 0;
+    margin: 0.5em 0;
```

Comment:
I cleaned up the other things mentioned but think we should leave this since it calculates the margin as half the font size for each header which is what we want. The alternative would be to manually calculate that and add it as pixel values to each individual rule.